tests/ui/specs/Popup/Popup-specs.js

```diff
@@ -730,9 +748,10 @@ describe('Popup', function () {
 				Page.waitForOpen(popup);
 
 				expectOpen(popupCommon);
+				Page.waitForFocused(popup.buttonOK);
 
-				Page.spotlightDown(); // OK button must have focus after Popup opens. This should be fixed and this line removed.
 				Page.spotlightRight();
+				Page.waitForFocused(popup.buttonCancel);
 				Page.spotlightDown();
 
 				expect(popup.buttonCancel.isFocused()).to.be.true();
```

Comment:
When I run the test manually, this test case is failed. The final focus is on the buttonPopup7.
The popup position is different from sandstone's the result is different.
Let's add a delay for the clarification.
```suggestion
				Page.spotlightDown();
				Page.waitForFocused(popupCommon.buttonPopup7);

				expect(popupCommon.buttonPopup7.isFocused()).to.be.true();
```